commit

```diff
@@ -1,32 +1,38 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
+if [ -z "${1:-}" ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
```

Comment:
On second thought, this and `set -euo pipefail` will break the empty `git commit` command which is useful when writing  a long commit message. It opens an editor for such commits when we don't provide the `-m` flag.